tests/sqllogictests/suites/query/02_function/02_0062_function_unnest

```diff
@@ -396,6 +396,36 @@ select count() from (select unnest(id), c1 from fold_count)
 ----
 4
 
+query T
+explain select count() from (select unnest(id), c1 from fold_count)
```

Comment:
explain should be inside explain folder.

Reply:
The explain results between standalone mode and cluster mode are different, should put this test into `standalone` folder.